### PR TITLE
Golf a few proofs

### PR DESCRIPTION
**Pull request by pitmonticone**

### CODE REVIEW

File: BonnAnalysis/Dual.lean [hunk 1]
```diff
@@ -40,27 +40,26 @@ structure IsConjExponent (p q : ℝ≥0∞) : Prop where
 namespace IsConjExponent
 
 lemma symm (hpq : p.IsConjExponent q) : q.IsConjExponent p := by
-    rw [isConjExponent_iff, add_comm, hpq.inv_add_inv_conj]
+  simp [isConjExponent_iff, add_comm, hpq.inv_add_inv_conj]
```
Comment (fpvandoorn) [inline BonnAnalysis/Dual.lean:43]: how is this an improvement? `rw`/`simp_rw` are preferred over `simp`.

Comment (pitmonticone) [reply BonnAnalysis/Dual.lean:43]: Ops, sorry. You're totally right.

Comment (pitmonticone) [reply BonnAnalysis/Dual.lean:43]: I'll change it as soon as I get access to my PC.

Comment (pitmonticone) [reply BonnAnalysis/Dual.lean]: Done

**Pull request merged**